test/src/runner/testRunnerChaiExpect.js

```diff
@@ -48,7 +48,7 @@ describe('testRunnerChaiExpect', function() {
 
         assert.ok(ex.message.startsWith('expected [ { ELEMENT: \'0\' } ] to have a length of 2 but got 1'));
 
-        assert.strictEqual(runner.results.modules.sampleWithChai.tests, 2);
+        assert.strictEqual(runner.results.modules.sampleWithChai.tests, 4);
```

Comment:
I think we should maintain number of tests as it was before (not considering hooks as separate tests)

Reply:
agreed and fixed.